client/executor/wasmi/src/lib.rs

```diff
@@ -561,14 +595,17 @@ impl WasmRuntime for WasmiRuntime {
 				error!(target: "wasm-executor", "snapshot restoration failed: {}", e);
 				e
 			})?;
-		call_in_wasm_module(ext, &self.instance, method, data, &self.host_functions)
+		call_in_wasm_module(
+			ext, &self.instance, method, data, &self.host_functions, self.enable_stub,
```

Comment:
Please put each parameter it its own line.

Reply:
done